worker/dbaccessor/worker.go

```diff
@@ -294,23 +330,241 @@ func (w *dbWorker) initializeDqlite() error {
 	return nil
 }
 
-func (w *dbWorker) openDatabases() error {
-	db, err := w.dbApp.Open(context.TODO(), "controller")
+func (w *dbWorker) openDatabase(namespace string) (coredatabase.TrackedDB, error) {
+	dbWorker, err := w.cfg.NewDBWorker(w.dbApp, namespace, WithClock(w.cfg.Clock), WithLogger(w.cfg.Logger))
 	if err != nil {
-		return errors.Trace(err)
+		return nil, errors.Trace(err)
 	}
 
-	w.dbHandles["controller"] = db
-	return nil
+	if err := w.dbRunner.StartWorker(namespace, func() (worker.Worker, error) {
```

Comment:
This changes the prior behaviour, and allows a consumer to create any database arbitrarily.

Reply:
Which is what we'd want, unless you'd like to restrict that for now?

For example, we would want this functionality when creating a new model.

Reply:
As we discussed. Future behaviour will be to:
- Open the controller DB, read the model list and open model DBs.
- Upon GetDB for a non-cached namespace, read the DBs again and verify that the model exists.

This way we don't have mess about with a watcher or 3rd-party worker.